samples/certs/generate-workload.sh

```diff
@@ -15,17 +15,26 @@
 #   limitations under the License.
 
 name=${1:-foo}
-san="spiffe://trust-domain-$name/ns/$name/sa/$name"
+ns=${2:-$name}
+sa=${3:-$name}
+dir=${4:-"."}
+tmp=${5:-""}
+san="spiffe://trust-domain-$name/ns/$ns/sa/$sa"
```

Comment:
Print out an "echo" for SAN for potential debugging?

Reply:
echo in line 54